karma.conf.js

```diff
@@ -123,6 +127,19 @@ module.exports = function(config) {
       });
     }
 
+    if (options['native-iterator']) {
+      customLaunchers = {
+        SL_Chrome: {
+          base: 'SauceLabs',
+          browserName: 'chrome'
+        },
+        SL_Firefox: {
+          base: 'SauceLabs',
+          browserName: 'firefox'
+        }
```

Comment:
Did you try setting the browser `version` property like in https://docs.saucelabs.com/reference/test-configuration/#version-browser-